docs/SCTValidation.md

```diff
@@ -85,10 +91,11 @@ get an SCT which covers a precursor version of the certificate, known as a
 **precertificate**.
 
 This is conceptually just the final certificate without the embedded SCT list,
-but there's a technical difficulty: CAs are only supposed to issue a single
-version of a given valid certificate, and are not supposed to allow duplicates.
-If a CA signed the same certificate both with and without the embedded SCT list,
-this would be in violation of this rule.
+but there's a technical difficulty: CAs are supposed to ensure that certificates
+have a serial number that is unique for a given issuing key.  If a CA signed the
+same certificate both with and without the embedded SCT list, this would be in
```

Comment:
"this would" -> "some people felt that this might"
(most didn't :))

Reply:
Fixed.
(Plus I reinstated "valid", which got lost in the previous edits and is needed for the next graf to make sense)